pkg/multiTenancyPlugins/naming/nameScoping.go

```diff
@@ -248,3 +272,24 @@ func getContainerId(container *cluster.Container, tenantId string, containerRefe
 	}
 	return "", errors.New(CONTAINER_REFERENCE_NOT_FOR_CONTAINER_INFO)
 }
+
+// send request to next handler in record mode.  When it returns remove traces of tenant id in response
+func doReq(nameScoping *DefaultNameScopingImpl, command utils.CommandEnum, cluster cluster.Cluster, w http.ResponseWriter, r *http.Request, swarmHandler http.Handler) error {
+	newReq, e1 := utils.ModifyRequest(r, nil, "", "")
```

Comment:
This is a hold over from following Doron's modifyRequest utility to modify the request.
I will investigate to figure out how to slim down the create volume to its bear mininum.